test/uk/gov/hmrc/customs/financials/emailthrottler/controllers/EmailThrottlerControllerSpec.scala

```diff
@@ -46,7 +46,6 @@ class EmailThrottlerControllerSpec extends SpecBase {
         ),
```

Comment:
Could you please split line no 74 as it is very long line? Better to split after =